### PR TITLE
fix publishing to provide proper RIDs

### PR DESCRIPTION
**Pull request by anurse**
also ensures that transitive project restores use the correct frameworks

/cc @davidfowl @Tratcher @pakrym @JunTaoLuo @BrennanConroy

### CODE REVIEW

File: src/Microsoft.Dnx.Tooling/Publish/PublishPackage.cs [hunk 1]
```diff
@@ -6,9 +6,11 @@
 using Microsoft.Dnx.Runtime;
 using Microsoft.Extensions.PlatformAbstractions;
 using NuGet;
+using System.Diagnostics;
```
Comment (Tratcher) [inline src/Microsoft.Dnx.Tooling/Publish/PublishPackage.cs]: nit: ordering

File: src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public PublishProject(ProjectDescription projectDescription)
         public string WwwRoot { get; set; }
         public string WwwRootOut { get; set; }
         public bool IsPackage { get; private set; }
-
+
```
Comment (Tratcher) [inline src/Microsoft.Dnx.Tooling/Publish/PublishProject.cs]: nit

**Comment by Tratcher** [comment]
@muratg

**Comment by davidfowl** [comment]
When will the RID madness end?

**Comment by anurse** [comment]
Fixing travis now.

**Comment by anurse** [comment]
Should be fixed now, there was a problem with how I was parsing the runtime names.

**Comment by JunTaoLuo** [comment]
looks good :shipit:

**Comment by anurse** [comment]
Alright, one travis worked. I'm going to squash and merge, then monitor the CI

**Pull request merged**